src/sagemaker/session.py

```diff
@@ -132,6 +132,10 @@ def upload_data(self, path, bucket=None, key_prefix='data'):
                 creates it).
             key_prefix (str): Optional S3 object key name prefix (default: 'data'). S3 uses the prefix to
                 create a directory structure for the bucket content that it display in the S3 console.
+            extra_args (dict): Optional extra arguments that may be passed to the upload operation.Similar to
+                ExtraArgs parameter in S3 upload_file function.Please refer to the ExtraArgs parameter
+                documentation here :
```

Comment:
a couple nitpicks: add spaces after the periods and remove the space after "here"

Reply:
Thanks @laurenyu .Fixed the spaces.